### PR TITLE
Address codacy stuff

### PR DESCRIPTION
**Pull request by michael-mclawhorn**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: luthier/src/main/lua/wikiApp/tables.lua [hunk 1]
```diff
@@ -119,6 +119,7 @@ M.physical = {
         }
     },
     air = {
+        type = "permissive",
```
Comment (archolewa) [inline luthier/src/main/lua/wikiApp/tables.lua]: What does this have to do with codacy?

File: luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Copyright 2019 Oath Inc.
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
-package com.yahoo.bard.webservice.config.luthier
+package com.yahoo.bard.webservice.data.config.luthier;
```
Comment (archolewa) [inline luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy]: You fixed the package, but not the file location (should be under `webservice/data/config/luthier`).

File: luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy [hunk 3]
```diff
@@ -68,4 +78,71 @@ class LuthierIndustrialParkSpec extends Specification {
             secondTestDimension.is(testDimension)
             ! differentTestDimension.is(testDimension)
     }
+
+    def "When a metric name is fetched the second time, it refers to the same object as the first one"() {
```
Comment (kevinhinterlong) [inline luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialParkSpec.groovy]: Typo? Looks like it was copied from the previous test
```suggestion
    def "When a metric name is fetched, it refers to the same object as the original definition"() {
```

File: luthier/src/main/lua/wikiApp/tables.lua (hunk 1 above)
Comment (michael-mclawhorn) [reply luthier/src/main/lua/wikiApp/tables.lua]: It increases test coverage.
